### ISSUE TITLE
Getting errors on client build. 

### ISSUE THREAD

**Issue by ddaza** [webpack/webpack-dev-server#367]
Updated to webpack 
```
├── webpack@1.12.11
├── webpack-core@0.6.8
├── webpack-dev-middleware@1.4.0
├── webpack-dev-server@1.14.1

```

I am seeing an error: 

```
Uncaught TypeError: Cannot read property 'replace' of null
```

in line 5 of `webpack-dev-server/client/index.js` : 

```javascript
var scriptHost = scriptElements[scriptElements.length-1].getAttribute("src").replace(/\/[^\/]+$/, "");
```

**Comment by globexdesigns** [comment]
+1

**Comment by zag2art** [comment]
+1

**Comment by cloudsan** [comment]
+1, rollback to @1.14.0 is fine for me,

**Comment by csaintc** [comment]
:+1:

**Comment by le0pard** [comment]
The same error after update to 1.4.1

**Comment by liujason** [comment]
+1 rolledback to @1.14.0 fixed the problem.

**Comment by jaylinski** [comment]
also getting this error on client when adding an inline-script-tag

**Comment by Stas404** [comment]
+1 hi all.

**Comment by kwtangerine** [comment]
+1. thanks for the tip, rolling back to 1.14.0 works for me too

**Comment by christophehurpeau** [comment]
:+1:

**Comment by kenjim83** [comment]
+1 me to.  1.14.0 works.

**Comment by Qrysto** [comment]
+1, same issue as this and #381. Why is this unresolved for so long?

**Comment by goodseller** [comment]
+1

**Comment by jaylinski** [comment]
@sokra can you have a quick look at this? there are already two issues and three PRs open for this problem.

**Comment by EwanValentine** [comment]
+1

**Comment by jakubzitny** [comment]
+1

**Comment by denich** [comment]
+1

**Comment by foxtree** [comment]
+1

**Comment by incleaf** [comment]
+1

**Comment by numical** [comment]
+1

**Comment by saponifi3d** [comment]
+1

**Comment by EvNaverniouk** [comment]
For those who want to make the leap to Webpack 2.0 beta, this issue appears to be fixed in webpack-dev-server `2.1.0-beta.0`

**Comment by SpaceK33z** [comment]
Dupe of #117.
